### PR TITLE
Proofread README.md

### PR DESCRIPTION
**Pull request by fractaledmind**
Just a collection of fixes for small typos and minor grammatical infelicities.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ Turn a fresh Ubuntu installation into a fully-configured, beautiful, and modern
...
 
-Nothing in Omakub provides solutions to problems you couldn't also solve a million other ways. The main benefit is in The Omakase Spirit. The idea that an entire setup experience can benefit from being tailored upfront by someone with strong opinions about what works and looks good together. This doesn't make the choices necessarily better than other choices. Linux has inspired a million options for a million tastes. That's great and worthy of celebration. But there's a large constituency of developers who are more than willing to trade ultimate bespoke customization for a cohesive package of goods, at least until they understand what all the options are and have fully bought into making the switch to Linux.
```
Comment (silva96) [inline README.md:11]: This change is a typo rigth?

File: README.md [hunk 2]
```diff
@@ -4,11 +4,11 @@ Turn a fresh Ubuntu installation into a fully-configured, beautiful, and modern
...
+Nothing in Omakub provides solutions to problems you couldn't also solve a million other ways. The main benefit is in The Omakase Spirit—the idea that an entire setup experience can benefit from being tailored upfront by someone with strong opinions about what works and looks good together. This doesn't make the choices necessarily better than other choices. Linux has inspired a million options for a million tastes. That's great and worthy of celebration. But there's a large constituency of developers who are more than willing to trade ultimate bespoke customization for a cohesive package of goods, at least until they understand what all the options are and have fully bought into making the switch to Linux.
```
Comment (jeremy) [inline README.md:11]: Purposeful period. Keep original.

File: README.md [hunk 3]
```diff
@@ -4,11 +4,11 @@ Turn a fresh Ubuntu installation into a fully-configured, beautiful, and modern
...
 
-This isn't a project for someone already versed in the intricasies of nixOS or relishing a fresh install of Arch. It's using vanilla Ubuntu because that's one of the most widely adopted Linux distributions, and one that is even a pre-install option from many computer vendors. But while Ubuntu has a great package manager in apt, many of the tools that developers want either haven't been packaged, need more recent versions than what has been frozen in the LTS, or need actions post-install necessary for the best operation. Omakub includes all those scripts needed.
+This isn't a project for someone already versed in the intricasies of nixOS or who relishes a fresh install of Arch. It's using vanilla Ubuntu because that's one of the most widely adopted Linux distributions, and one that is even a pre-install option from many computer vendors. But while Ubuntu has a great package manager in apt, many of the tools that developers want either haven't been packaged, need more recent versions than what has been frozen in the LTS, or need actions post-install necessary for the best operation. Omakub includes all those scripts needed.
```
[the remark this answers is not in the document]
Comment (fractaledmind) [reply README.md:7]: Good spelling catch. Missed that. For the sentence, I don't quite parse the intended grammar. Removing the first phrase, the sentence (as I read it) becomes: "This isn't a project for someone relishing ..." Why a present participle—the person is presently relishing installing Arch? The first clause is generally descriptive of the type of person. I tried to mirror that structure.

File: README.md (hunk 2 above)
Comment (fractaledmind) [reply README.md:11]: Understood. My first read through I got tripped up by the structural sentence that isn't a grammatical sentence, but I am one for a stylistic flourish here and there, so happy to oblige.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jeremy) [reply README.md:7]: Splitting "someone or who" makes the revision more awkward, however.

**Comment by fractaledmind** [comment]
@jeremy thanks for the good conversation. I will proof-read the site repo and close this PR 👍

**Pull request closed without merging**